ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -107,6 +107,19 @@ describe("Form with File Input - Basic Tests", () => {
                 checkHTML(id, field.getModel().getState())
         });
     })
+    it(" after attaching the file widget value is reset to allow duplicate file ", () => {
+        const sampleFileName = 'sample2.txt', fileInput = "input[name='fileinput1']";
+        cy.get(fileInput).should('have.value', "");
+        cy.get(fileInput).attachFile(sampleFileName).then(() => {
+            cy.get(".cmp-adaptiveform-fileinput__filename").contains(sampleFileName);
+            cy.get(fileInput).should(($element) => {
```

Comment:
Can you please attach duplicate file again, thats the use-case we need to test since this is an e2e test and not a unit test.

Reply:
Added the test cases for attaching the same file again. Please review.